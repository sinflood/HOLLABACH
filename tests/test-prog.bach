comp Banjo {
loop 2 {
[D2/4 R F2/4 R G2/4 R R D2/4]
[ R F2/4 R A2/4 G2/4 R R R]
[D2/4 R F2/4 R G2/4 R R F2/4]
[ R D2/4 R R R R R R]
}
measureLen=2
cool=[ G2/1 D2/1 G2/1 D2/1 G2/1 D2/1 G2/1 D2/1]
cool
}

comp Clarinet {
[B2/1]
[B2/4]
}

comp somethinge {
[R B2/4]
[B2/4 R]
}
