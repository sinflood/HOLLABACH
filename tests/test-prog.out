Timing Resolution (pulses per quarter note)
4

Instrument,120,,Instrument,71,,Instrument,105,,

Tick,Note(0-127),Velocity(0-127),Tick,Note(0-127),Velocity(0-127),Tick,Note(0-127),Velocity(0-127),
0,38,1
4,41,1
8,43,1
14,38,1
18,41,1
22,45,1
24,43,1
32,38,1
36,41,1
40,43,1
46,41,1
50,38,1
64,38,1
68,41,1
72,43,1
78,38,1
82,41,1
86,45,1
88,43,1
96,38,1
100,41,1
104,43,1
110,41,1
114,38,1
128,43,4
129,38,4
130,43,4
131,38,4
132,43,4
133,38,4
134,43,4
135,38,4
136,43,4
137,38,4
138,43,4
139,38,4
140,43,4
141,38,4
142,43,4
143,38,4
,,,0,47,4
,,,8,47,1
,,,,,,4,47,1
,,,,,,8,47,1
